src/components/Settings/Settings.js

```diff
@@ -257,24 +258,13 @@ const Settings = ( ) => {
         <View className="mb-9">
           <Heading4>{t( "INATURALIST-MODE" )}</Heading4>
           <View className="mt-[22px]">
-            <RadioButtonRow
+            <SwitchRow
+              testID="advanced-interface-switch"
               smallLabel
-              checked={isDefaultMode}
-              onPress={( ) => {
-                setIsDefaultMode( true );
-                setIsAllAddObsOptionsMode( false );
-              }}
-              label={t( "Default--interface-mode" )}
-            />
-          </View>
-          <View className="mt-4">
-            <RadioButtonRow
-              testID="advanced-interface-option"
-              smallLabel
-              checked={!isDefaultMode}
-              onPress={( ) => {
-                setIsDefaultMode( false );
-                setIsAllAddObsOptionsMode( true );
+              value={!isDefaultMode}
+              onValueChange={newValue => {
```

Comment:
We're not diligent about doing this everywhere in our codebase, but in general in React it's preferred not to add functions in the render function, since this function will get recreated every time the component re-renders. My two cents is that these mild performance implications add up. One thing we've gotten feedback on in Seek and earlier versions of iNat Next is that it "doesn't feel native" because of performance, and I think that's something that will become more of a headache down the road if we don't think about it while we're coding now.